src/parser/deathknight/shared/runeforges/RuneOfHysteria.tsx

```diff
@@ -98,9 +106,9 @@ class RuneOfHysteria extends Analyzer {
         size="flexible"
         tooltip={(
           <>
-            You gained {Math.floor(this.runicPowerGainedByHysteria)} RP by using {SPELLS.RUNE_OF_HYSTERIA.name}, resulting in {this.gainedDeathStrikes} additional Death Strikes.<br />
+            You gained {Math.floor(this.runicPowerGainedByHysteria)} RP by using {SPELLS.RUNE_OF_HYSTERIA.name}, resulting in {this.gainedDeathStrikes} additional {SPELLS.DEATH_STRIKE.name}.<br />
```

Comment:
Is it possible to make the mention of DS conditional on if the players spec is blood?  Occurred to me that someone playing hysteria on frost or unholy might take this tooltip the wrong way

Reply:
No one cares about the other specs anyways 😄 
But you're right, DS hint is now only shown when the player is blood. Could add other spells for frost/unholy, for now its just showing `Gained x RP and wasted X RP` for them